aws-opentelemetry-distro/src/amazon/opentelemetry/distro/exporter/otlp/aws/metrics/otlp_aws_emf_exporter.py

```diff
@@ -314,7 +254,8 @@ def _convert_sum(self, metric, dp) -> Tuple[Any, int]:
         record.attributes = dp.attributes
 
         # For Sum, set the sum_data
-        record.value = dp.value
+        record.sum_data = type('SumData', (), {})()
```

Comment:
nit: Rename `SumData` to `Sum`, or rename `Histogram/ExpHistogram` to `HistogramData/ExpHistogramData`